scripts/serve/serve-cli.js

```diff
@@ -0,0 +1,30 @@
+const commander = require('commander');
+const path = require('path');
+const serve = require('./serve');
+
+const packageJson = require('../../package.json');
+const defaultWebpackConfig = require('./defaultWebpackConfig');
+
+
+// Parse process arguments
+commander
+  .version(packageJson.version)
+  .option('--config <path>', 'The webpack config to serve.')
```

Comment:
I think we could simplify this to be:
```diff
-const defaultWebpackConfig = require('./defaultWebpackConfig');
...
// provide default config path as the default to load
-.option('--config <path>', 'The webpack config to serve.')
+.option('--config <path>', 'The webpack config to serve.', './defaultWebpackConfig')

-.option('--port <n>', 'The port the server should listen on.', parseInt)
+.option('--port <n>', 'The port the server should listen on.', 8080)
....
// drop if(commander.config) because serve will execute the config function

serve({
-   config,
+  config: require(path.resolve(commander.config)),
})

```

Reply:
^^^Ignore my above suggestion, I see the difference now. We should change `./defaultWebpackConfig` to './loadWebpackConfig' to clarify the webpack loading pattern implemented

Reply:
Yeah, i realized how bad my name was after your mistake.